addons/website_sale/static/tests/tours/website_free_delivery.js

```diff
@@ -49,3 +49,28 @@ registry.category("web_tour.tours").add('check_free_delivery', {
             run: function () {}, // it's a check
         }
     ]});
+
+    registry.category("web_tour.tours").add('check_errored_delivery', {
```

Comment:
I think you need to extract this tour in the separate file with the corresponding name, `website_free_delivery` does not suit.